src/js/select2/dropdown/attachBody.js

```diff
@@ -32,6 +32,15 @@ define([
           self._resizeDropdown();
         });
       }
+
+	  function repositionOnSelectionChange() {
+	    if (!this.options.get('closeOnSelect')) {
```

Comment:
I was originally trying to only reposition if I knew the dropdown wasn't about to be closed due to the closeOnSelect option.  I had originally incorrectly assumed the container.isOpen might have given me the wrong value depending on the order of the decorator chain, however, it does look like this check works more generically.  I'll modify and commit.